package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>redf</name>
+  <version>0.0.1</version>
```

Comment:
https://github.com/osrf/redf/blob/813fd1fcbb21d6e82381c29dcce77c9ef0c3c3ca/Cargo.toml#L3

We are technically at `0.1.0`

Reply:
Nice catch! I've updated the version here.